tests/lib/rules/strict-transport-security/tests.ts

```diff
@@ -24,7 +24,7 @@ const requestJSONAsyncMock = (responseObject) => {
         isDataURI() {
             return false;
         },
-        normalizeString(string) {
+        normalizeString(string = '') {
```

Comment:
can you change `string` to `str` or anything else, we are using TypeScript and in a first look `string` can look as a type instead of a variable

Reply:
@molant just this detail. the rest for me is ok to merge

Reply:
Done, you will have to squash as I've done the change directly on GitHub (in the middle of something else on my laptop)